app/boards/shields/nice_view/nice_view.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+&nice_view_spi {
+	status = "okay";
+	cs-gpios = <&pro_micro 1 GPIO_ACTIVE_HIGH>;
+
+	nice_view: ls0xx@0 {
+		compatible = "sharp,ls0xx";
+		label = "DISPLAY";
+		spi-max-frequency = <1000000>;
+		reg = <0>;
+		width = <160>;
+		height = <68>;
+	};
+};
+
+&pro_micro_i2c {
```

Comment:
This ties the shield to only working with PM based boards. Given XIAO gaining popularity, I wonder if this should be in `shields/nice_view/boards/*.overlay` files for all the existing PM boards?